sys-boot/gnu_efi/gnu_efi-3.0.4.recipe

```diff
@@ -0,0 +1,31 @@
+SUMMARY="GNU EFI Library - Kernel support"
+DESCRIPTION="
+Develop EFI applications for ARM-64, ARM-32, x86_64, IA-64 (IPF), and IA-32 (x86)
+platforms using the GNU toolchain and the EFI development environment.
+"
+HOMEPAGE="https://sourceforge.net/projects/gnu-efi/"
+COPYRIGHT="
+	1999-2007 Hewlett-Packard Co.
+	2006-2010 Intel Co.
+	"
+LICENSE="BSD (3-clause)"
+SOURCE_URI="http://downloads.sourceforge.net/project/gnu-efi/gnu-efi-$portVersion.tar.bz2"
```

Comment:
It is recommended to drop the "/project/" directory in the URI in order to allow the redirector service to find the tarball shouldin case upstream moves it to a different sub-directory under /files/.
```
SOURCE_URI="https://downloads.sourceforge.net/gnu-efi/gnu-efi-$portVersion.tar.bz2" 
```
I'm also suggesting https (instead of http) because I just noticed that SourceForge no longer redirects https to http, That's cool!

BTW, you can also replace "sourceforge" by "sf" if you wish to shorten SOURCE_URI a little, but that's not required.